examples/messenger-built-in-nlp/index.js

```diff
@@ -0,0 +1,29 @@
+const { MessengerBot } = require('bottender');
+const { createServer } = require('bottender/express');
+
+const bot = new MessengerBot({
+  accessToken: '__FILL_YOUR_TOKEN_HERE__',
+  appSecret: '__FILL_YOUR_SECRET_HERE__',
+});
+
+bot.onEvent(async context => {
+  /* eslint-disable no-console */
```

Comment:
You can use `// eslint-disable-next-line no-console` for the next line
or `// eslint-disable-line no-console` after console.log.

Do not disable the rule in the whole file.

Reply:
That rule should be off from `examples/.eslintrc.js`
https://github.com/Yoctol/bottender/blob/master/examples/.eslintrc.js

Reply:
Ok~ I removed this line